ValkeyJSON.md

```diff
@@ -859,6 +866,13 @@ Info metrics are visible through the “info json” or “info modules” comma
 | json.max-document-size | 64            | MB   | Maximum memory allowed for a single JSON document.    |
 | josn.max-path-limit    | 128           |      | Maximum nesting levels within a single JSON document. |
 
+TBD:
+
+We are considering a review feedback of not enforcing document size limit, because RedisJSON doesn't have the limit and Valkey 
+core data types don't have limits either. There could be two options:
+1. Make the default value of json.max-document-size 0 or -1, meaning unlimited.
+2. Remove the config.
```

Comment:
Can you also move this out of the current section, it's supposed to be its own section in the doc.

```suggestion:

## Open Questions

We are considering a review feedback of not enforcing document size limit, because RedisJSON doesn't have the limit and Valkey 
core data types don't have limits either. There could be two options:
1. Make the default value of json.max-document-size 0 or -1, meaning unlimited.
2. Remove the config.
```

Reply:
updated